Dutch/main/MiuiGallery.apk/res/values-nl/plurals.xml

```diff
@@ -67,7 +67,7 @@
         <item quantity="one">%1$d</item>
     </plurals>
     <plurals name="how_many_pics">
-        <item quantity="other">"%d foto's"</item>
+        <item quantity="other">"%d foto's "</item>
```

Comment:
Zou iemand hier naar kunnen kijken?
Als je in de galerij helemaal naar beneden scrolt kan je deze string zien. Op telefoon staat het als %dFoto's. Aan elkaar en met hoofdletter dus. Ik begrijp het niet helemaal.